javascript/javascript1/week3/warmup_exercise/script.js

```diff
@@ -0,0 +1,83 @@
+//ITEM ARRAY REMOVAL
+
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+
+for (let i = 0; i < names.length; i++) {
+  if (names[i] === nameToRemove) {
+    names.splice(i, 1);
+  }
+}
+
+console.log(names);
+
+//WHEN WILL WE BE THERE?
+
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432,
+};
+
+function calculateTime(speed, destinationDistance) {
+  let time = destinationDistance / speed;
+  const hours = Math.floor(time);
+  const minutes = Math.round((time % 1) * 60);
+  return `${hours} hours ${minutes} minutes`;
```

Comment:
If the travel time was just one minute, you would return "0 hours and 1 minutes". Can you think of a way to improve this? 😄

Reply:
Good thought :) I have updated my code to display as 1 minute/ 1 hour(instead of hours).